easybuild/toolchains/compiler/craype.py

```diff
@@ -137,6 +137,7 @@ class CrayPEGCC(CrayPECompiler):
     def __init__(self, *args, **kwargs):
         """CrayPEGCC constructor."""
         super(CrayPEGCC, self).__init__(*args, **kwargs)
+        self.COMPILER_UNIQUE_OPTION_MAP.update({'openmp': 'fopenmp'})
```

Comment:
@gppezzi I would go with this instead, just to match the style below:

```python
self.COMPILER_UNIQUE_OPTION_MAP['openmp'] = 'fopenmp'
```

other than that: should be good to go